script/lint_darwin_plugins.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# This script lints and tests iOS and macOS platform code.
+
+# So that users can run this script from anywhere and it will work as expected.
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+REPO_DIR="$(dirname "$SCRIPT_DIR")"
+
+source "$SCRIPT_DIR/common.sh"
+
+function lint_package() {
+  local package_name=$1
+  local platform=$2
+  local podspec_dir="$REPO_DIR/packages/$package_name/$platform/$package_name.podspec"
+  local failure_count=0
+
+  if [[ -f "$podspec_dir" ]]; then
+    echo "Linting $platform $package_name.podspec"
+
+    # Build as frameworks.
+    # This will also run any tests set up as a test_spec. See https://blog.cocoapods.org/CocoaPods-1.3.0.
+    pod lib lint "$podspec_dir" --allow-warnings --fail-fast --silent
+    if [[ $? -ne 0 ]]; then
```

Comment:
nit: Please quote the variables here and throughout.

https://google.github.io/styleguide/shell.xml?showone=Quoting#Quoting

Reply:
Thanks, I forgot to check that.  Hopefully I got them all.